[2023-11-03T18:13:59.406+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: people.create_people_table manual__2023-11-03T18:13:57.594148+00:00 [queued]>
[2023-11-03T18:13:59.421+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: people.create_people_table manual__2023-11-03T18:13:57.594148+00:00 [queued]>
[2023-11-03T18:13:59.421+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-11-03T18:13:59.422+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-11-03T18:13:59.423+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-11-03T18:13:59.441+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_people_table> on 2023-11-03 18:13:57.594148+00:00
[2023-11-03T18:13:59.450+0000] {standard_task_runner.py:55} INFO - Started process 535 to run task
[2023-11-03T18:13:59.451+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'people', 'create_people_table', 'manual__2023-11-03T18:13:57.594148+00:00', '--job-id', '104', '--raw', '--subdir', 'DAGS_FOLDER/people.py', '--cfg-path', '/tmp/tmpxwax5jtq']
[2023-11-03T18:13:59.455+0000] {standard_task_runner.py:83} INFO - Job 104: Subtask create_people_table
[2023-11-03T18:13:59.540+0000] {task_command.py:376} INFO - Running <TaskInstance: people.create_people_table manual__2023-11-03T18:13:57.594148+00:00 [running]> on host 6b65bc8eaea5
[2023-11-03T18:13:59.636+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=people
AIRFLOW_CTX_TASK_ID=create_people_table
AIRFLOW_CTX_EXECUTION_DATE=2023-11-03T18:13:57.594148+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-03T18:13:57.594148+00:00
[2023-11-03T18:13:59.650+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-11-03T18:13:59.655+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS people (
                User_Id varchar(100)  primary key,
                First_Name varchar(100),
                Last_Name varchar(100),
                Sex varchar(100),
                Email varchar(100)
            );
        , parameters: None
[2023-11-03T18:13:59.658+0000] {postgres.py:96} INFO - NOTICE:  relation "people" already exists, skipping

[2023-11-03T18:13:59.670+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=people, task_id=create_people_table, execution_date=20231103T181357, start_date=20231103T181359, end_date=20231103T181359
[2023-11-03T18:13:59.711+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-11-03T18:13:59.740+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
